app/Http/Controllers/Api/ContestsController.php

```diff
@@ -22,10 +25,17 @@ public function __construct()
     }
 
     /**
-     * Get a collection of waiting rooms.
+     * Get a collection of contests
      */
-    public function index()
+    public function index(Request $request)
     {
+        $run_nid = $request->query('run_nid');
```

Comment:
If you're only using the `$run_nid` in one place, no need for saving it as a variable.

(also can we maintain the naming we use elsewhere and just call it `$campaign_run_id`)